package.json

```diff
@@ -28,17 +29,19 @@
     "uuid": "^3.2.1"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "api": "nodemon server/server.js --exec babel-node --presets es2015,stage-2",
+    "start": "concurrently \"npm run api\" \"npm run start-client\"",
+    "start-client": "HTTPS=true react-scripts start",
+    "api": "BABEL_DISABLE_CACHE=1 nodemon --exec babel-node -r dotenv/config server/server.js --presets @babel/preset-env",
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject"
   },
   "devDependencies": {
-    "babel-cli": "^6.26.0",
-    "babel-preset-es2015": "^6.24.1",
-    "babel-preset-stage-2": "^6.24.1",
-    "nodemon": "^1.18.7",
+    "@babel/core": "^7.14.5",
+    "@babel/node": "^7.14.5",
+    "@babel/preset-env": "^7.14.5",
```

Comment:
Babel v6 has issues with using dotenv in both determining path to package and also permeating env vars to child modules.